pkg/web/http.go

```diff
@@ -275,7 +275,7 @@ func Run(
 		"/system/search",
 		authMiddleware(http.HandlerFunc(c.search), auth))
 
-	addr := fmt.Sprintf("%s:%d", cfg.Web.Host, cfg.Web.Port)
```

Comment:
This is bad. It renders the web-host config useless.

Reply:
You are right. I am handling this now differently.